Index: src/main/java/com/cn/piaware/storm/topology/SimpleStormTopology.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/cn/piaware/storm/topology/SimpleStormTopology.java	(revision 34994a6860ea9f07e5d07be1073511814dd60adf)
+++ src/main/java/com/cn/piaware/storm/topology/SimpleStormTopology.java	(revision )
@@ -34,9 +34,9 @@
         LocalCluster cluster = new LocalCluster();
         cluster.submitTopology("simple-topology", config, topology);
 
-        Utils.sleep(60000); // run for 1 minute
+        //Utils.sleep(60000); // run for 1 minute
         //Utils.sleep(300000); // run for 5 minutes
-        //Utils.sleep(600000); // run for 10 minutes
+        Utils.sleep(600000); // run for 10 minutes
         cluster.killTopology("simple-topology");
         cluster.shutdown();
     }
Index: src/main/java/com/cn/piaware/storm/topology/PiAwareTopology.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/cn/piaware/storm/topology/PiAwareTopology.java	(revision 34994a6860ea9f07e5d07be1073511814dd60adf)
+++ src/main/java/com/cn/piaware/storm/topology/PiAwareTopology.java	(revision )
@@ -1,6 +1,11 @@
 package com.cn.piaware.storm.topology;
 
+import java.util.Enumeration;
 import java.util.Iterator;
+import java.util.Properties;
+import java.util.Collections;
+import java.util.List;
+import java.util.ArrayList;
 
 import org.apache.commons.configuration2.CompositeConfiguration;
 import org.apache.commons.configuration2.PropertiesConfiguration;
@@ -8,11 +13,32 @@
 
 public class PiAwareTopology {
     public static void main(String[] args) {
+
+        /*
+         * 1. print properties in alphbetical order
+         */
+        Properties props = System.getProperties();
+        Enumeration<Object> ek = props.keys();
+        Collections.sort(ek);
+
+        while (ek.hasMoreElements()) {
+            String key = ek.nextElement().toString();
+            String val = props.get(key).toString();
+            System.out.println(key + "=" + val);
+        }
+
+        List<Object> ll = new ArrayList(ek);
+
+
+        /*
+         * 2. read properties file from correct place in Maven File Directory Structure
+         */
         CompositeConfiguration config = new CompositeConfiguration();
 
         //config.addConfiguration(new PropertiesConfiguration("piaware.properties"));
-        config.addConfiguration(new PropertiesConfiguration(""));
+        //config.addConfiguration(new PropertiesConfiguration(""));
 
+        /*
         Iterator<String> keys = config.getKeys();
 
         String kk = keys.toString();
@@ -22,6 +48,7 @@
             String val = config.getProperty(key).toString();
             System.out.println(key + " = " + val);
         }
+        */
 
     }
 }
